cluster/cluster_members.mapper.go

```diff
@@ -17,57 +17,57 @@ import (
 var _ = api.ServerEnvironment{}
 
 var internalClusterMemberObjects = RegisterStmt(`
```

Comment:
I think those vars require a name update too?

Reply:
Those vars are still called `internal`. Should they be moved to start with `core` too? There are some functions too.

Reply:
Ah oops, I put the changes in the wrong commit, fixed now :)